shell/client/shell-client.js

```diff
@@ -909,6 +909,10 @@ Router.map(function () {
   this.route("account", {
     path: "/account",
 
+    waitOn() {
+      return globalSubs;
```

Comment:
+1

It's almost too bad we don't have a GlobalSubsHelper of some kind specific to the Sandstorm codebase.